indico/modules/categories/controllers/management.py

```diff
@@ -117,6 +122,33 @@ def _process(self):
                                                     icon_form=icon_form, logo_form=logo_form)
 
 
+class RHAPIEventMoveRequests(RHManageCategoryBase):
+    def _process_GET(self):
+        return EventMoveRequestSchema(many=True).jsonify(self.category.event_move_requests)
+
+    @use_kwargs({
+        'state': EnumField(MoveRequestState, required=True)
+    })
+    def _process_POST(self, state):
+        move_requests = parser.parse({
+            'requests': EventRequestList(required=True, category=self.category, validate=not_empty)
+        }, unknown=EXCLUDE)['requests']
+
+        if state not in (MoveRequestState.accepted, MoveRequestState.rejected):
```

Comment:
Maybe taking an `accept` bool arg or an `action` string would be better? Especially since the logic of actually approving/rejecting the request should most likely be in an operation function which then also does stuff like sending notification emails and adding something to the event log.

Reply:
Sounds good, I agree. In the beginning, this wasn't very clear. 
I'm assuming this logging would only apply to rejects since approving already shows the default move log, unless we want to show it differently?